frontend/src/scenes/session-recordings/player/metaLogic.ts

```diff
@@ -0,0 +1,63 @@
+import { kea } from 'kea'
+import { metaLogicType } from './metaLogicType'
+import { sessionRecordingLogic } from 'scenes/session-recordings/sessionRecordingLogic'
+import { sessionRecordingPlayerLogic } from 'scenes/session-recordings/player/sessionRecordingPlayerLogic'
+import { eventWithTime } from 'rrweb/typings/types'
+import { PersonType } from '~/types'
+
+const getPersonProperties = (person: Partial<PersonType>, keys: string[]): string | null => {
+    if (keys.some((k) => !person?.properties?.[k])) {
+        return null
+    }
+    return keys.map((k) => person?.properties?.[k]).join(', ')
+}
+
+export const metaLogic = kea<metaLogicType>({
+    path: ['scenes', 'session-recordings', 'player', 'metaLogic'],
+    connect: {
+        values: [
+            sessionRecordingLogic,
+            ['sessionPlayerData'],
+            sessionRecordingPlayerLogic,
+            ['snapshots', 'time', 'scale', 'meta'],
+        ],
+    },
+    selectors: {
+        sessionPerson: [
+            (selectors) => [selectors.sessionPlayerData],
+            (playerData): Partial<PersonType> => {
+                return playerData?.person
+            },
+        ],
+        description: [
+            (selectors) => [selectors.sessionPerson],
+            (person) => {
+                const location = getPersonProperties(person, ['$geoip_city_name', '$geoip_country_code'])
```

Comment:
We should probably handle the case where we don't have these properties. (e.g. they don't have the geo plugin running)

Reply:
Yup I already handle this by returning `null` in `getPersonProperties` if both properties don't exist on `person.properties` and omitting null values in line 37.

Reply:
Ah nice! Missed that!